packages/eyes/src/index.ts

```diff
@@ -1,16 +1,43 @@
-/* tslint:disable:no-console */
+require('dotenv').config();
```

Comment:
I think it should be the project's responsibility to populate environment variables. We shouldn't expect that the project uses `.env`, and should simply read `process.env`